modules/oe_subscriptions_digest/tests/src/Functional/UserDigestTest.php

```diff
@@ -0,0 +1,191 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\Tests\oe_subscriptions_digest\Functional;
+
+use Drupal\field\Entity\FieldConfig;
+use Drupal\message_digest\Entity\MessageDigestInterval;
+use Drupal\Tests\BrowserTestBase;
+use Drupal\Tests\flag\Traits\FlagCreateTrait;
+use Drupal\user\UserInterface;
+
+/**
+ * Tests the user subscriptions page.
+ */
+class UserDigestTest extends BrowserTestBase {
```

Comment:
Make a base class like the others. 
For me:
* we make a base class that contains the test methods, and it requires implementing classes to provide a few methods;
* we make a base class without any test method but only the `doX` methods, and we leave the implementations of the class to add test methods. This is what we did for the other tests.

It's up to you, normally the best scenario would be point 1.